hyperspace/core/src/command.rs

```diff
@@ -304,8 +307,7 @@ impl QueryCmd {
 		let config = self.parse_config().await?;
 		let chain = config.into_client().await?;
 
-		chain.query_channels().await?;
-
+		let channels = chain.query_channels().await?.iter().for_each(|c| println!("{}: {}", c.0, c.1));
```

Comment:
Nit: you can deconstruct c
https://doc.rust-lang.org/rust-by-example/flow_control/match/destructuring/destructure_structures.html